1-exercises/E-for-loop/exercise1.js

```diff
@@ -4,11 +4,14 @@
     Change the while loop below into a for loop.
 */
 
-
 // Change the below code to use a for loop instead of a while loop.
-let i = 0;
-while(i < 26) {
-    console.log(String.fromCharCode(97 + i));
-    i++;
-}
+// let i = 0;
+// while (i < 26) {
+//     console.log(String.fromCharCode(97 + i));
+//     i++;
+// }
 // The output shouldn't change.
+
+for (let i = 0; i < 26; i++) {
+    console.log(String.fromCharCode(65 + i), String.fromCharCode(97 + i));
+}
```

Comment:
Good work - adding upper case letters wasn't necessary, but it's good to see that you are interested in how fromCharCode works.